src/groups/bmq/bmqp/bmqp_protocol.h

```diff
@@ -4075,7 +4078,8 @@ MessagePropertyHeader::setPropertyNameLength(int value)
     // PRECONDITIONS
     BSLS_ASSERT_SAFE(value >= 0 && value <= k_MAX_PROPERTY_NAME_LENGTH);
 
-    d_reservedAndPropNameLen = value & k_PROP_NAME_LEN_MASK;
```

Comment:
In this case, k_MAX_PROPERTY_NAME_LENGTH corresponds to 4095 so this may need a closer look